HDSAviz/Interactive_vis_toolkit.ipynb

```diff
@@ -0,0 +1,443 @@
+    "\n",
+    "Basically files should be organized as : \n",
+    "\n",
+    "C: HDSAviz_data/\n",
+    "C: HDSAviz_data/analysis_CO.txt \n",
+    "C: HDSAviz_data/analysis_CO2.txt \n",
+    "etc.\n",
+    "\n",
+    "C: HDSAviz_data/interactivevislib.py \n",
+    "C: HDSAviz_data/Interactive_vis_toolkit.ipynb\n",
+    "\n",
+    "\"\"\"\n",
+    "\n",
+    "#import Basic_Plot  once Basic_Plot.py file is ammended.\n",
+    "\n",
+    "import os\n",
+    "from bokeh.models.widgets import Panel, Tabs\n",
+    "from bokeh.io import output_file, output_notebook, show, vform\n",
+    "from ipywidgets import FloatSlider\n",
+    "from IPython.html.widgets import interact\n",
+    "from collections import OrderedDict\n",
+    "from bokeh.plotting import figure, show, output_notebook, output_file\n",
+    "\n",
+    "# initializing the range of values for total order (ST) and first order \n",
+    "# (S1) sensitivity tests \n",
+    "ST_Range = 1\n",
+    "S1_Range = 1\n",
+    "\n",
+    "def change_directory():\n",
+    "    \"\"\"\n",
+    "    This function changes the working directory from C:Users/UserName to\n",
+    "    C:/HDSAviz_data/ which is chosen as convention for current project.\n",
+    "    \n",
+    "    Output: \n",
+    "    Changes working directory and prints current directory\n",
+    "    \"\"\"\n",
+    "    path = \"c:/HDSAviz_data\"\n",
+    "    os.chdir(path)\n",
+    "    print 'current working directory: ' + os.getcwd()\n",
+    "\n",
+    "\n",
+    "def gather_all_analysis_files():\n",
+    "    \"\"\"\n",
+    "    This function gathers all the analysis files from C:/HDSAvis_data/ \n",
+    "    \n",
+    "    Output: \n",
+    "    Returns a list of analysis files that can be fed into plotting function\n",
+    "    \n",
+    "    \"\"\"\n",
+    "    new = !ls\n",
+    "    analysis_files = []\n",
+    "    for files in new:\n",
+    "        if files.startswith(\"analysis\"):\n",
+    "            analysis_files.append(files)\n",
+    "    print 'all analysis files gathered from current directory!' \n",
+    "    return analysis_files\n",
+    "\n",
+    "def plot_all_outcomes_burtin(S1_Range,ST_Range): \n",
+    "    \"\"\"\n",
+    "    This function plots the burtin type plots for all the outcome \n",
+    "    variables in different text files.\n",
+    "    \n",
+    "    Input :\n",
+    "    S1_Range: Minimum value of sensitivity for first order effects * 0.001, \n",
+    "    ST_Range: Minimum value of sensitivity for total order effects * 0.001\n",
+    "    \n",
+    "    Output:\n",
+    "    Tab based plotting of Graphs. \n",
+    "    \"\"\"\n",
+    "    working_dir = !pwd\n",
+    "    if working_dir != 'C:/HDSAviz_data/' :\n",
+    "        change_directory()\n",
+    "    \n",
+    "    analysis_files = gather_all_analysis_files()\n",
+    "    tabs_dictionary = {}\n",
+    "    \n",
+    "    for i,files in zip(range(len(analysis_files)), analysis_files):\n",
```

Comment:
Thanks for the note. 

Making codes compatible is in my to do list. 
I will remove unnecessary functions like this getting files function since its already there but currently my code wasn't compatible with Basic_Plot_Updated.py and thus i couldn't make this change for now....

To Do List 4: